hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/protocol/SCMSecurityProtocol.java

```diff
@@ -77,4 +80,16 @@ String getOMCertificate(OzoneManagerDetailsProto omDetails,
    */
   String getCACertificate() throws IOException;
 
+  /**
+   * Get list of certificates meet the query criteria.
+   *
+   * @param type            - node type: OM/SCM/DN.
+   * @param startSerialId   - start certificate serial id.
+   * @param count           - max number of certificates returned in a batch.
+   * @param isRevoked       - whether list for revoked certs only.
+   * @return list of PEM encoded certificate strings.
+   */
+  List<String> listCertificate(HddsProtos.NodeType type, long startSerialId,
```

Comment:
Nit: getCertificates may be a better name here
```suggestion
  List<String> getCertificates(HddsProtos.NodeType type, long startSerialId,
```

Reply:
list is chosen here instead of get to be consistent with similar API/CLI for pipeline/container/blocks.